src/azul/indexer/document.py

```diff
@@ -403,9 +407,22 @@ def translate_fields(cls,
                         # None value (and then further translate that None value
                         # according to the field type) so ES doesn't discard it.
                         doc = [None]
-                    return [field_type.to_index(value) for value in doc]
+                    try:
```

Comment:
The point of the FieldType abstraction is so we don't have to code for special cases in this method.